src/components/notes/add.jsx

```diff
@@ -1,14 +1,16 @@
 import React, { useState, useCallback } from 'react'
 
-import { withClient } from 'cozy-client'
+import { useClient } from 'cozy-client'
 
 import Button from 'cozy-ui/react/Button'
-import { translate } from 'cozy-ui/react/I18n'
+import { useI18n } from 'cozy-ui/react/I18n'
 import BarButton from 'cozy-ui/react/BarButton'
 
 import { createNoteDocument, generateReturnUrlToNotesIndex } from 'lib/utils'
 
-const Add = ({ t, className, client }) => {
```

Comment:
I see that t and client were removed from props, but I don't see the call site to see if those props were removed ? Was it wrapped in withClient or translate before ?

Reply:
yep, `t` and `client` where from withClient() and translate() 

See https://github.com/cozy/cozy-notes/pull/125/files#diff-e70fe36241ca5b019a08946e7d13bfe3L42

Reply:
Ah ok, sorry, since the export default had changed of place, I did not connect the dots.